### DISCUSSION TITLE
Invalid character in statusline!

### DISCUSSION THREAD

**Discussion by aiyolo** [vscode-neovim/vscode-neovim#2313] [General]
I have already set JetBrainsMono NF font in editor, why some characters can't be rendered？
![image](https://github.com/user-attachments/assets/ec245c57-ad14-4814-9b0b-5e1c4ec9672e)

**Comment by xiyaowong** [comment]
ui font ≠ editor font

**Reply by aiyolo** [reply]
So how can I fix it?

**Reply by xiyaowong** [reply]
There is currently no official way to change the font of the ui. https://github.com/microsoft/vscode/issues/519
